crates/miden-lib/asm/miden/input_note.masm

```diff
@@ -22,15 +22,20 @@ use.miden::note
 #! Invocation: exec
 export.get_assets_info
     # start padding the stack
-    push.0.0 movup.2
-    # => [note_index, 0, 0]
+    push.0 swap
+    # => [note_index, 0]
+
+    # push the flag indicating that we want to request assets info from the note with the specified
+    # index
+    push.0
+    # => [0, note_index, 0]
 
     exec.kernel_proc_offsets::input_note_get_assets_info_offset
-    # => [offset, note_index, 0, 0]
+    # => [offset, 0, note_index, 0]
```

Comment:
I think it would be clearer if we wrote `is_current_note` or `is_current_note = 0` in the stack comment instead of just `0`.